htdocs/class/libraries/vendor/xoops/xmf/src/Xmf/Metagen.php

```diff
@@ -451,8 +451,8 @@ protected static function purifyText($text, $keyword = false)
     {
         $myts = \MyTextSanitizer::getInstance();
         $text = str_replace('&nbsp;', ' ', $text);
-        $text = str_replace('<br />', ' ', $text);
-        $text = str_replace('<br/>', ' ', $text);
+        $text = str_replace('<br>', ' ', $text);
+        $text = str_replace('<br>', ' ', $text);
         $text = str_replace('<br', ' ', $text);
```

Comment:
This should not have been changed.
1) these lines are to help in cleaning random input, not generating output. They need to stay as they were.
2) this is part of Xmf, and is maintained separately so they should not be updated here (nothing in libraries should be)

Because of 2, we don't need to fix this, as it will be cleaned up later -- Xmf will be refreshed soon.